pkg/server/server.proto

```diff
@@ -23,6 +23,7 @@ service Runner {
     rpc CreateTestCase(TestCaseWithSuite) returns (HelloReply) {}
     rpc UpdateTestCase(TestCaseWithSuite) returns (HelloReply) {}
     rpc DeleteTestCase(TestCaseIdentity) returns (HelloReply) {}
+    rpc GetYaml(TestSuiteIdentity) returns (YamlData) {}
```

Comment:
Consider this is testSuite yaml. How about moving this line after line 16? And rename it to `GetTestSuiteYaml`.

Reply:
Thanks for your review, I have corrected